presets/4.3/tune/karate_freestle.txt

```diff
@@ -0,0 +1,87 @@
+Karate FS tune
+#$ TITLE: Karate Freestyle
```

Comment:
Maybe add "Karate Freestyle **5''**" for the title.
I figured we need to specify more which this or that tune is suitable for.

Reply:
Easy sorry for the mistakes 😂 I’ll fix it up when I get home

Reply:
good point